website/docs/usefullpaths.md

```diff
@@ -1,12 +1,20 @@
-usefullpaths - This page was auto-generated. Feel free to help us improve the documentation by creating a pull request.
+Turn on/off full paths usage in diagnostics
 
 ```lua
-usefullpaths (value)
+usefullpaths "switch"
 ```
 
+By default, the generated project files will use the compilers default settings, which is in most cases "On" for debug and "Off" for release.
+In Visual Studio, this overrides the /FC flag which is forced on when using debug builds.
+
 ### Parameters ###
 
-`value` - needs documentation.
+`switch` is an identifier.
+
+| Option      | Availability                |
```

Comment:
Instead of availability, these should be descriptions. See `pic` docs, for example.